changelogs/fragments/add_bgp_af_rm.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
Does this deprecate eos_bgp module? If so, we need to update the module doc and runtime.yml.

Reply:
@NilashishC I shall take it up as a separate PR .